util/fusermount.c

```diff
@@ -1347,6 +1347,7 @@ static void usage(void)
 	       " -V		    print version\n"
 	       " -o opt[,opt...]    mount options\n"
```

Comment:
Btw, any reason why `-o` doesn't have a long version?  I like using long versions in bash scripts, as they are more self-documenting.

Reply:
Will do!

Reply:
https://github.com/libfuse/libfuse/pull/764